src/Controls/src/Core/LegacyLayouts/Layout.cs

```diff
@@ -503,7 +513,7 @@ internal virtual void OnChildMeasureInvalidated(VisualElement child, Invalidatio
 			int count = children.Count;
 			for (var index = 0; index < count; index++)
 			{
-				if (LogicalChildrenInternal[index] is VisualElement v && v.IsVisible && (!v.IsPlatformEnabled || !v.IsPlatformStateConsistent))
+				if (LogicalChildrenInternal[index] is VisualElement { IsVisible: true } v && (!v.IsPlatformEnabled || !v.IsPlatformStateConsistent))
```

Comment:
This line can very likely be removed thanks to the fact that `IsPlatformEnabled` is now being checked on `VisualElement` before triggering measure invalidation.